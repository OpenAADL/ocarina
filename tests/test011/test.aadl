--  test for number output

property set titi is
  Tata : aadlinteger applies to (all);
end titi;

thread Toto
  properties
    titi::Tata => (
      12, 0, 1E6, 123_456,
      12.0, 0.0, 0.456, 3.14159_26,
      2#1111_1111#, 16#FF#, 016#0ff#,
      16#e#E1, 2#1110_0000#,
      16#F.FF#E+02, 2#1.1111_1111_1110#E11,
      16#1E.0#E-2, 02#11.110#E-5,
      008#123.07#E1,
      -13#C.C#E+2,
      3#1.0000000000000000000000000000000001#E9,
      -3#1.1000100010001000100010001000100010001#E40,
      2#111#E30,
      9#123456780#E4
    );
end Toto;