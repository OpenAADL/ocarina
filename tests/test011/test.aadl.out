test.aadl:9:05: titi::Tata is a list while the corresponding property name at test.aadl:4:03 is not a list.
Cannot analyze AADL specifications
